orc8r/cloud/deploy/terraform/orc8r-aws/outputs.tf

```diff
@@ -11,6 +11,11 @@
 # limitations under the License.
 ################################################################################
 
+output "region" {
```

Comment:
Last time I did this ran into some weird issues. I would avoid doing this till we test this change if it is backported to 1.6.

Reply:
Good to know. Will revert